chainio/clients/avsregistry/bindings.go

```diff
@@ -209,6 +211,11 @@ func NewBindingsFromConfig(
 		if err != nil {
 			return nil, utils.WrapError("Failed to get AvsDirectory address", err)
 		}
+		allocationManagerAddr, err = contractServiceManager.AllocationManager(&bind.CallOpts{})
+		if err != nil {
+			return nil, utils.WrapError("Failed to get AllocationManager address", err)
+		}
+
 	}
 
```

Comment:
Here, we could add the same idea of the check as in elcontracts:
```
if isZeroAddress(cfg.AllocationManagerAddress) {
...
```

if we don't get the address of the allocation Manager, we should ignore it while creating the return of the function.